analysis.py

```diff
@@ -49,80 +99,143 @@ def optimal_angular_distribution_hyperparameters(train_data, save_dir, val_data,
     ion_data = []
     for i in range(num_clusters):
         ion_data.append(np.vstack((train_data[labels_train==i, 0:3], train_data[labels_train==i, 3:6])))
-
     val_ion_data = np.vstack((val_data[:,0:3], val_data[:,3:6]))
     val_ion_labels = np.hstack((labels_val, labels_val)).reshape(-1)
     
+    electron_data = []
+    for i in range(num_clusters):
+        electron_data.append(np.vstack((train_data[labels_train==i, 9:12], train_data[labels_train==i, 12:15])))
+
+    val_elec_data = np.vstack((val_data[:,9:12], val_data[:,12:15]))
+    val_elec_labels = np.hstack((labels_val, labels_val)).reshape(-1)
+    
     # Choose best angular distribution hyperparameters
     for L in range(0, L_max+1):
         for num_bins in bin_range:
             Bs = []
             for i in range(num_clusters):
                 B_lms, _ = fitting.fit_Y_lms_binning_least_squares(
-                    ion_data[i], L,
+                    electron_data[i], L,
```

Comment:
Here we see the change from ion to electron data.

Reply:
I might have been just trying stuff out and forgot about this change 🤦. We can keep this as ions or we can try using electrons and the neutral? i would think those two would have a bit less biased distribution due to the change into the molecular frame but I guess the best solution would be to make that a runtime parameter in the config file?